standard/classes.md

```diff
@@ -5416,3 +5418,52 @@ When the body of the async function terminates, the return task is moved out of
 If the return type of the async function is `void`, evaluation differs from the above in the following way: Because no task is returned, the function instead communicates completion and exceptions to the current thread’s ***synchronization context***. The exact definition of synchronization context is implementation-dependent, but is a representation of “where” the current thread is running. The synchronization context is notified when evaluation of a `void`-returning async function commences, completes successfully, or causes an uncaught exception to be thrown.
 
 This allows the context to keep track of how many `void`-returning async functions are running under it, and to decide how to propagate exceptions coming out of them.
+
+### §asynchronous-streams Asynchronous stream creation and usage
+
+A stream can be created and consumed asynchronously.
```

Comment:
The definition of “stream” needs to be added here, elsewhere in the Standard we have streams of Unicode characters and tokens, uses of `StreamReader` etc.

See previous comments for how this might be done; it may even be decided to move this section into the (now following) §15.15 Iterators and Async Streams clause.